pex/pex.py

```diff
@@ -444,7 +444,7 @@ def execute_pkg_resources(spec):
     else:
       # setuptools < 11.3
       runner = entry.load(require=False)
-    runner()
+    sys.exit(runner())
```

Comment:
this seems like a potentially awkward place to `sys.exit` - esp as potentially called by a library consumer. is there no better path to plumbing the return value/execution context and `sys.exit`'ing at the end of `PEX.execute()`?

at a minimum, moving this to either `PEX.execute_script()` or its calling branch in `PEX._execute()` would make slightly more sense to me given the context.